Remotion/Web/ClientScript/Scripts/SmartNavigation.ts

```diff
@@ -202,7 +201,7 @@ class SmartFocus
     return undefined as unknown as boolean;
   }
 
-  public static SetFocus (element: JQuery): void
+  private static SetFocus (element: HTMLElement): void
```

Comment:
have to keep that one public, at least for now. There's a downstream workaround based on being able to replace / override this function.
```suggestion
  public static SetFocus (element: HTMLElement): void
```